bootstrap/gamma/gamma_e2e_test.go

```diff
@@ -104,3 +104,28 @@ func TestGammaWithEmptyBlocks(t *testing.T) {
 	t.Run("Benchmark", testGammaWithEmptyBlocks(`{"transaction-pool-time-between-empty-blocks":"200ms"}`))
 	t.Run("LeanHelix", testGammaWithEmptyBlocks(fmt.Sprintf(`{"active-consensus-algo":%d,"transaction-pool-time-between-empty-blocks":"200ms"}`, consensus.CONSENSUS_ALGO_TYPE_LEAN_HELIX)))
 }
+
+func TestGammaSetBlockTime(t *testing.T) {
+	randomPort := test.RandomPort()
+	runMain(t, randomPort, "")
+	endpoint := fmt.Sprintf("http://0.0.0.0:%d", randomPort)
+
+	require.True(t, test.Eventually(WAIT_FOR_BLOCK_TIMEOUT, waitForBlock(endpoint, 1)))
+
+	res, err := http.Post(endpoint+"/debug/gamma/inc-time?seconds-to-add=10", "text/plain", nil)
+	require.NoError(t, err, "failed incrementing next block time")
+	require.EqualValues(t, 200, res.StatusCode, "http call to increment time failed")
```

Comment:
I may prefer `REST/API/RPC` over `http` here - since it's the "how" and not the "what"...

Reply:
can you please elaborate?

Reply:
it's a minor comment - the language draws the attention to the transport. but the transport did not necessarily failed here. the PRC/API did not go through - thats the point. we _know_ it by looking at the http code. but what it means is that the server failed to acknowledge the request. 

So possibly: `"failed incrementing next block time, with http code %v", res.StatusCode`